tests/csm/rest/test_iam_users.py

```diff
@@ -986,11 +1042,13 @@ def test_36582(self):
         self.log.info("##### Test started -  %s #####", test_case_name)
         self.log.info("[START] Testing that user can create Key pair for the I AM user using UID")
         self.log.info("Creating IAM user.")
-        user_id, display_name, access_key, secret_key = self.get_IAM_user_payload("keys")
+        user_id, display_name = self.get_IAM_user_payload()
```

Comment:
Is this correct?
as Doesnt looks to be part of purge change

Reply:
these were not getting used